src/styles/variables.scss

```
@@ -0,0 +1,8 @@
+$bc-color: #eee;
+$col-counts: 31;
+$border-size: 1px;
+$cal-padding: 10px;
+$limit-columns: 7;
+$gap: 1px;
+$days: ('mon' 0, 'tue' 1, 'wed' 2, 'thu' 3, 'fri' 4, 'sat' 5, 'sun' 6);
+$cal-size: 100px;
```

Do not abbreviate names to unreadable ones.

```suggestion
$background-color: #eee;
$calendar-counts: 31;
$border-size: 1px;
$calendar-padding: 10px;
$limit-columns: 7;
$gap: 1px;
$days: ('mon' 0, 'tue' 1, 'wed' 2, 'thu' 3, 'fri' 4, 'sat' 5, 'sun' 6);
$calendar-size: 100px;
```